vgcore_server

```diff
@@ -0,0 +1,14 @@
+Receiving data. 4bytes used.
```

Comment:
OH BAH EH

Reply:
Oh bah eh the missed paste :laughing:

Reply:
oh bah eh the debug file that doesn't have to be here because of the .gitignore